examples/integrate-with-next/src/pages/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { getGeneratedCode } from '../modelina';
+import Head from 'next/head';
+import styles from '@/styles/Home.module.css';
+
+function HomePage({ models }: any): any {
```

Comment:
any reason why this is any? I always question myself whenever I have to use any :D

Reply:
Nothing but laziness 😆

Reply:
I'd like to forbid the use of any in this repo, and only allow exceptions when there's no way around it 😆

Reply:
Thats fair, I just thought it didnt matter much for an example 😆 

Let me change it 👍

Reply:
that's true, but now you have a better example 😄

Reply:
That's true 😄